### DISCUSSION TITLE
finance.yahoo.com: nuisance

### DISCUSSION THREAD

**Discussion by rajaschava** [uBlockOrigin/uAssets#26615] [Q&A]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.). Comment in the old issue threads even when they are closed or even if you have a different problem.
- [X] I did not remove any of the [default filter lists](https://github.com/user-attachments/assets/1879fc88-09bc-4de4-9e2f-3834a6ab396a), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL address of the web page

`https://finance.yahoo.com/news/trump-poised-inherit-strong-economy-120023064.html`

### Category

nuisance

### Description

A small window with video is displayed at the bottom. There is X icon to close it. But when you scroll up or down it appears again!

### Other extensions used

facebook container
Anti adblock
clear urls

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

![image](https://github.com/user-attachments/assets/6974dfbe-bcd0-41d6-95c2-000ba16409f3)

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.61.2
Firefox: 133
filterset (summary):
 network: 142721
 cosmetic: 49812
 scriptlet: 21348
 html: 2231
listset (total-discarded, last-updated):
 default:
  user-filters: 3-1, never
  ublock-filters: 41014-131, now
  ublock-badware: 12530-1, now
  ublock-privacy: 1662-23, now
  ublock-unbreak: 2603-1, now
  ublock-quick-fixes: 180-8, now
  easylist: 78125-252, now
  easyprivacy: 53328-73, now
  urlhaus-1: 24745-0, now
  plowe-0: 3548-994, now
filterset (user): [array of 3 redacted]
trustedset:
 added: [array of 1 redacted]
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 1043 ms
 maxAssetCacheWait: 344 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 230
 network:
  yahoo.com: 146
  ay.delivery: 1
  brightcove.com: 44
  chartbeat.net: 3
  clean.gg: 1
  doubleclick.net: 1
  imasdk.googleapis.com: 2
  scorecardresearch.com: 13
  spot.im: 16
  taboola.com: 1
  yimg.com: 2
 extended:
  ##.sdaContainer
  ##+js(set-constant, Navigator.prototype.globalPrivacyControl, fa…
  ##+js(set-constant, navigator.globalPrivacyControl, false)
```
</details>

**Comment by krystian3w** [comment]
In uBo MV2 **test**:

```adb
finance.yahoo.com##.evp-main[class*=" "][style*=" "]
```

IMO not fixed yet in EasyList Annoyances (module "Other") to cover uBo Lite (MV3) too.
